src/TelemetryWrapper.ts

```diff
@@ -60,6 +69,10 @@ export module TelemetryWrapper {
         }
     }
 
+    export function currentSession() {
```

Comment:
Currently, the session and reporter are in one class. They have different concerns and roles so I would suggest that we separate them.

Reply:
@akaroml If I got it right, they're not defined in one class.
Session is defined in `Session.ts`
TelemetryReporter is defined in package `vscode-extension-telemetry`.

`TelemetryWrapper.ts` defines static variables and functions, e.g. a static TelemetryReporter instance,  and a static method `currentSession()` .

Reply:
got that, `Session` and `TelemetryReporter` decoupled.